### PR TITLE
Add abess

### PR DESCRIPTION
**Pull request by Mamba413**
A machine learning software using cibuildwheel. cibuildwheel is excellent and very helpful!

### CODE REVIEW

File: docs/data/projects.yml [hunk 1]
```diff
@@ -30,8 +36,8 @@
 
 - name: Parselmouth
   gh: YannickJadoul/Parselmouth
-  pypi: praat-parselmouth
-  notes: A Python interface to the Praat software package, using pybind11, C++17 and CMake, with the core Praat static library built only once and shared between wheels.
+  pypi: praat-parselmouthA Python interface to the Praat software package, using pybind
+  notes: 11, C++17 and CMake, with the core Praat static library built only once and shared between wheels.
```
Comment (joerick) [inline docs/data/projects.yml:40]: Could these changes be reverted please?

Comment (Mamba413) [reply docs/data/projects.yml:40]: I have not made these changes, and I believe they can be reverted.

Comment (joerick) [reply docs/data/projects.yml:40]: Would you mind reverting them? They're in your PR, so I suppose that you made them by accident.

Comment (Mamba413) [reply docs/data/projects.yml:40]: No problem. I going to revert them.

Comment (Mamba413) [reply docs/data/projects.yml]: I believe they have been reverted. Can you check it?

Comment (henryiii) [reply docs/data/projects.yml]: LGTM

**Pull request merged**